src/pages/index.tsx

```diff
@@ -8,21 +10,54 @@ import task from '@/interfaces/task.type';
 import Accordion from '@/components/Accordion';
 
 const TASKS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks`;
+const SELF_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/users/self`;
 
-function renderCardList(tasks: task[]) {
-  return tasks.map((item: task) => <Card content={item} key={item.id} />);
+async function updateCardContent(cardDetails: any) {
```

Comment:
You can import the type/interface for tasks from [here](https://github.com/Real-Dev-Squad/website-status/blob/develop/src/interfaces/task.type.ts)